content/en/docs/concepts/services-networking/service.md

```diff
@@ -184,6 +184,19 @@ An ExternalName Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
+### Endpoint Slices
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+Endpoint Slices are an API resource that can provide a more scalable alternative
+to Endpoints. Although conceptually quite similar to Endpoints, Endpoint Slices
```

Comment:
@robscott , Is it important to include the Endpoint Slices section in the Service page?
Or, is it reasonable to add a link to the new Endpoint slices page in the `Whats Next` section?

Reply:
I think eventually we should separate both Endpoints and Endpoint Slices from the Service page, but since Endpoints are featured so prominently there, and we're trying to replace them with Endpoint Slices, it seemed like it was necessary to have at least some reference to Endpoint Slices in the same context as Endpoints. I did just add a link to Endpoint Slices in What's Next on the Service page. If the combination of everything feels like it's too much, let me know. It might be worth creating a follow up issue to extract some of the Endpoints specific documentation from this page to somewhere else now that we have 2 options.

Reply:
👍